src/server/qtquick/private/wquickcoordmapper_p.h

```diff
@@ -65,8 +65,7 @@ class WQuickCoordMapper : public QQuickItem
 
 private:
     void updatePosition();
-
-    WQuickObserver *m_target;
+    WQuickObserver *target() const;
```

Comment:
Don't add a new function in header file.

Reply:
Use `WQuickObserverPrivate::get(...)->target` replace, like as QQuickItemPrivate::get function.